bin/fixtures/redis-debug.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+add_action( 'wp_footer', function(){
```

Comment:
It looks like the thinking here was that it is easier to put redis debug info in the footer than it is to add complexity to behat that would get behat reading the output of `terminus wp`, yes? Can you added a comment explaining what is happening, something like

```
This plugin is placed in mu-plugins by the testing scripts. Once in mu-plugins it will be loaded on all requests to WordPress and add Redis debug information to to the footer of the site when a GET param is present. Behat can then examine this debug output.
```